scanner/cmd/scanner/main.go

```diff
@@ -87,7 +87,11 @@ func main() {
 	}
 
 	ctx = zlog.ContextWithValues(ctx, "component", "main")
-	zlog.Info(ctx).Str("version", version.Version).Str("build_flavor", buildinfo.BuildFlavor).Msg("starting scanner")
+	zlog.Info(ctx).
+		Str("version", version.Version).
+		Str("vulnerability_version", version.VulnerabilityVersion).
```

Comment:
this is fine, but the only oddity is saying "vulnerability_version" in the Indexer, which doesn't track vulnerabilities. I don't think it's worth the effort coming up with a "catch-all" term, especially when the variable we use is "VulnerabilityVersion"

Reply:
Actually I also realized this is not quite right: https://github.com/stackrox/stackrox/blob/4.7.2/scanner/config/config.go#L179

The version we use may be `dev` which is not captured by `version.VulnerabilityVersion`. Perhaps we can expose the versions returned by that function

Reply:
Hrm.. the intent was to show the version embedded in the binary to inform future issues (ie: potential build issues) - the other values can be determined by looking at the embedded value + external config map - admittedly that intent was a reaction to a recent issue.

That is a good point regarding indexer showing vulnerability_version. I'll look to adjust this to print the values of each variable and the final URL - think may be better / more actionable.